### ISSUE TITLE
Loot notification

### ISSUE THREAD

**Issue by Skogstjur** [LightsHope/server#125]
Items register in chat that they are received but gold isn't.
Gold is received in inventory but not noted in chat.

Tried logging out and in, didn't change it.

Settings is not changed

**Comment by jsb** [comment]
I think that's blizzlike. By default, you didn't get any chat notification for looting gold unless you were in a group.

[Source](https://youtu.be/5_ZPdLEprMg?t=36m9s)

**Comment by Cabro** [comment]
@jsb - I call your claim false, as you can just toggle the Money loot announcement in chat options. He probably didn't like to get spammed with the "You have looted 1 silver." messages.

Right click on respective tab > Other messages > Money Loot

**Comment by Skogstjur** [comment]
@Cabro "Settings is not changed" It is standard settings as claimed before.
I was not in a group, I was looting mobs I just had killed.

**Comment by Cabro** [comment]
@Skogstjur yes i wasn't contesting your say, i was contesting the movie linked.

**Comment by GuybrushGit** [comment]
Tested and the packets being sent are the same in each case. Most likely a client issue.
